project/ScalafixBuild.scala

```diff
@@ -124,6 +124,11 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     lazy val buildInfoSettingsForRules: Seq[Def.Setting[_]] = Seq(
       buildInfoObject := "RulesBuildInfo"
     )
+
+    lazy val scalatestDeps = Def.setting {
```

Comment:
looks like you return a single element in both cases, so remove the `Seq` and rename to `scalatestDep` ?